tests/ami/scripts/test_rpm.sh.j2

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env bash
+# Copyright 2014-2020 Scalyr Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+set -e
+
+{% if verbose -%}
+set -x
+{% endif -%}
+
+# Give it some time to finish provisioning process. It's possible that we can SSH in before
+# cloud-init fully finished
+sleep 20
+
+uname -a
+cat /etc/*-release
+
+sudo yum install -y wget
+sudo yum install -y "{{python_package}}"
+
+# Install Sclayr agent.
+{% if install_package["type"] in ("file", "url") -%}
+{# type of the package is 'file' or 'url', that means that file
+should be uploaded by script or already uploaded by Libloud. #}
+    {% if install_package["type"] == "url" -%}
+    wget -O install_package.deb "{{ install_package["source"] }}"
+    {% elif upgrade_package["type"] == "file" -%}
```

Comment:
Same here, I assume it should be install_package["type"].

Or we could remove this line all together since it's a no-op anyway.